app/assets/stylesheets/blacklight/_facets.scss

```diff
@@ -36,12 +36,12 @@
 
 .facet-values {
   display: table;
+  table-layout: fixed;
```

Comment:
This is a different approach, but still has the problem of needing a fixed width on the "columns". This sets the count column larger to accommodate larger numbers.